crates/scheduler/src/utils.rs

```diff
@@ -251,9 +251,12 @@ pub fn get_process_tracker_id<'a>(
     runner: storage::ProcessTrackerRunner,
     task_name: &'a str,
     txn_id: &'a str,
```

Comment:
Please rename `txn_id` to `entity_id` or `object_id`, whatever you think makes more sense.